package.json

```
@@ -5,7 +5,7 @@
   "author": "Fuel Labs <contact@fuel.sh> (https://fuel.network/)",
   "private": true,
   "engines": {
-    "node": "^18.18.2",
+    "node": ">=18 <21",
```

Shouldn't it be like this?

```suggestion
    "node": ">=18.18.2 <21",
```

---

Absolutely